frontend/src/pages/notebookController/NotebookController.scss

```diff
@@ -56,6 +56,9 @@
         white-space: nowrap;
       }
     }
+    &-popover-help-icon {
```

Comment:
We should avoid creating more custom CSS styles. I have attached the possible solution to avoid using `Button`.